core/services/cable_guy/html/index.html

```diff
@@ -150,7 +150,7 @@
         this.userInputChanged = false
         console.debug(`Sending: ${JSON.stringify(data)}`)
         axios
-          .post('/v1.0/ethernet', data)
+          .post('/ethernet', data)
```

Comment:
keep in mind that doing so we are not going to be able to have version control over our API.

Reply:
I can take it back, no prob.

Reply:
I'm thinking if it makes sense, though.

Say we do a minor change, which would move 1.0 to 1.1. With the `/v1.0` on the query, we end up breaking compatibility when in fact it was not broke.

Or you're thinking in maintaining several API versions running concurrently (no problem on the technical aspect, OpenAPI support's that, but veeery difficult to maintain).

Reply:
Nonethless, I've just added version information to the API itself, so if the service using the API wants to somehow use that version information to handle errors, it can do it accessing .info.version on OpenAPI's json (/openapi.json).